worker-sys/src/types/hyperdrive.rs

```diff
@@ -0,0 +1,26 @@
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(extends=js_sys::Object)]
+    #[derive(Clone, PartialEq, Eq)]
+    pub type Hyperdrive;
+
+    #[wasm_bindgen(method, getter, js_name=connectionString)]
+    pub fn connection_string(this: &Hyperdrive) -> String;
+
+    #[wasm_bindgen(method, getter, js_name=host)]
+    pub fn host(this: &Hyperdrive) -> String;
+
+    #[wasm_bindgen(method, getter, js_name=port)]
+    pub fn port(this: &Hyperdrive) -> u16;
```

Comment:
On the JS side this is a `number` (https://github.com/cloudflare/workerd/blob/a1055534197598c01b9f8b2d0d4d8f7e27de13c5/types/defines/hyperdrive.d.ts#L32). Can I access it as a `u16` on the Rust side?

Reply:
I'm pretty sure the answer is yes, except for the larger integer types that map to BigInt.